app/assets/stylesheets/section.scss

```diff
@@ -0,0 +1,7 @@
+.section {
+  margin-bottom: 2rem;
```

Comment:
Could you use `px` for units pls. I'm open to being told that `rem` is better, but everything else is `px` atm so I'd like to keep it consistent until/unless we change it everywhere.